### ISSUE TITLE
Bug Report: setActorVelocity and setActorAcceleration on sprite cause game to crash

### ISSUE THREAD

**Issue by relma2** [KadeArchive/Kade-Engine#1534]
### If you are playing a downloaded version of the game, what operating system are you using?
Windows

## What version of the game are you using? Look in the bottom left corner of the main menu.
1.5.4

## Have you identified any steps to reproduce the bug? If so, please describe them below. Use images if possible.
uhhh.... use the `setActorVelocity` or setActorAcceleration` x/y methods on an flx sprite

## Solution?
This is because **FlxSprite**s do NOT have velocity or acceleration attributes, according to the [documentation](https://api.haxeflixel.com/flixel/FlxSprite.html)
So you cant actually access them the way the ModchartState.hx code does:
```
Lua_helper.add_callback(lua, "setActorVelocityX", function(x:Int, id:String)   {
    // SPRITES DONT FUCKING HAVE THIS
    getActorByName(id).velocity.x = x;
});
```

Upon looking at the FlxSprite code itself, theres an interface called IFlxSprite that _does_ have velocity/acceleration attributes, but the code to make the sprite move according to that physics isnt present in the base class

...you know im starting to think this might be a bug in Haxe Flixel

**Comment by Kade-github** [comment]
Can you check if you get this on 1.6?

**Comment by relma2** [comment]
I... do not, actually!
the issue was fixed on 1.6
i bashed out this quick code on dadbattle
```lua
function beatHit(beat)
    if (beat >= 80 and beat % 8 == 0) then
        setActorAccelerationX(-50, "dad")
    elseif (beat >= 80 and beat % 4 == 0) then
        setActorAccelerationX(50, "dad")
    end
end
```
and both velocity and acceleration work on Flxsprites like the dad character.
Well, thanks for the fix, i guess 😅
